<ui version="4.0" >
 <class>XpSetter</class>
 <widget class="QMainWindow" name="XpSetter" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QFrame" name="frame" >
    <property name="geometry" >
     <rect>
      <x>460</x>
      <y>10</y>
      <width>271</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="Ymin" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>190</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="Xslide" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>290</y>
       <width>25</width>
       <height>121</height>
      </rect>
     </property>
     <property name="singleStep" >
      <number>1</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval" >
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Zstep" >
     <property name="geometry" >
      <rect>
       <x>210</x>
       <y>250</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Zstep&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
    <widget class="QPushButton" name="Run" >
     <property name="geometry" >
      <rect>
       <x>170</x>
       <y>450</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Run</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11" >
     <property name="geometry" >
      <rect>
       <x>160</x>
       <y>140</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Y</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ScanBox" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>10</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Zmax" >
     <property name="geometry" >
      <rect>
       <x>200</x>
       <y>220</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinY" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>420</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals" >
      <number>4</number>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="Counter" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>40</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="suffix" >
      <string/>
     </property>
     <property name="prefix" >
      <string/>
     </property>
     <property name="maximum" >
      <number>1000000</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="Zname" >
     <property name="geometry" >
      <rect>
       <x>210</x>
       <y>160</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text" >
      <string>samz</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Ystep" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>250</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>190</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Min</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Mv" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>450</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Move</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>40</y>
       <width>33</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Count</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15" >
     <property name="geometry" >
      <rect>
       <x>220</x>
       <y>140</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Z</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Xmax" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>220</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>10</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Scan Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Xname" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>160</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text" >
      <string>samx</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>170</y>
       <width>33</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Name</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinZ" >
     <property name="geometry" >
      <rect>
       <x>200</x>
       <y>420</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals" >
      <number>4</number>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinX" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>420</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
     <property name="decimals" >
      <number>4</number>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Zmin" >
     <property name="geometry" >
      <rect>
       <x>200</x>
       <y>190</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>140</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>250</y>
       <width>33</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Steps</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Xstep" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>250</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="Yslide" >
     <property name="geometry" >
      <rect>
       <x>150</x>
       <y>290</y>
       <width>25</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval" >
      <number>10</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Xmin" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>190</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="Zslide" >
     <property name="geometry" >
      <rect>
       <x>220</x>
       <y>290</y>
       <width>25</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval" >
      <number>10</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Ymax" >
     <property name="geometry" >
      <rect>
       <x>130</x>
       <y>220</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>210</y>
       <width>22</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>420</y>
       <width>45</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Yname" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>160</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text" >
      <string>samy</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Elements</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>441</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
